app/views/users/edit.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="container">
+		<div class="setting">
+			<center><h3>Setting</h3></center>
+
+			<%= form_tag(user_path,{method:"put"})do %>
+				<div class="form-group">
+					<p>User Name</p>
+					<%= text_field_tag :name, @name, class: 'form-control' %>
```

Comment:
@yukoro Since you passed `@user` variable from `users#edit`, then update `@name` to `@user.name`